### ISSUE TITLE
remove a single split

### ISSUE THREAD

**Issue by josegpt** [project-repo/cagebreak#21]
Hello,

I was wondering if there is a way to `remove` a single split. I looked into the manual, only to find `only` which remove all splits.

Thanks and cagebreak is awesome!

**Comment by project-repo** [comment]
Yes, that is correct, there is currently no way to do this. So how
exactly would you imagine this to work? How would one choose which split
to remove?
cheers
project-repo

**Comment by josegpt** [comment]
I would imagine it to work like in Emacs or ratpoison in order to delete a split; it must be `focused` first.

**Comment by project-repo** [comment]
Huh interesting... I wasn't aware of the fact that ratpoison supported
this. We'll look into it and get back to you then! Again, as in issue
#20, unfortunately it may be a while until we
are able to implement this feature. However, thanks for bringing it up!
cheers
project-repo

**Comment by VehementHam** [comment]
Well I have tried using resize mode to fill the gap. The problem is, that I can get it most of the way there, but not the full length.

**Comment by project-repo** [comment]
This is implemented in 3.0.0 (see the documentation for `mergeleft`, etc.), closing.
